tests/conftest.py

```
@@ -3,6 +3,25 @@
 
 import pytest
 
+from sdrf_pipelines.ols.ols import OLS_AVAILABLE
+
+
+def pytest_configure(config):
+    """Register custom markers."""
+    config.addinivalue_line("markers", "ontology: tests that require ontology dependencies")
+
+
+def pytest_collection_modifyitems(config, items):
```

Ruff (0.14.14)

[warning] Unused function argument: `config`

(ARG001)